enter-env.sh

```diff
@@ -81,8 +82,26 @@ UserKnownHostsFile $scriptroot/morph-network/known_hosts
 BatchMode yes
 IdentitiesOnly yes
 IdentityFile $SSH_IDENTITY_FILE
+Include $SSH_CONFIG_FILE.inc
 EOF
 
+nix-instantiate --eval --json -E "let net = import $scriptroot/morph-network; in
+builtins.concatStringsSep \"\n\"
+  (map
+    (p: ''
+      Host \${p} eval-\${builtins.substring 17 10 p}
```

Comment:
`17` is the length of `ofborg-evaluator-` and `10` is an arbitrary number of characters following that. If we ever get to the point of having more than 10 digits-worth of evaluators, I think we'll be having larger problems than not being able to (easily) SSH into the ones after `9999999999`.